modules/platforms/cpp/core/include/ignite/compute/compute_func.h

```diff
@@ -57,6 +68,31 @@ namespace ignite
              * @return Computation result.
              */
             virtual R Call() = 0;
+
+        protected:
+            /*
+             * Get environment pointer.
+             *
+             * @return Env pointer.
+             */
+            ignite::impl::IgniteEnvironment* GetEnv()
```

Comment:
`IgniteEnvironment` is internal class, we should never show it to user. We should keep and return `Ignite` instance here.

Reply:
Return pointer to Ignite instead of env.